core/scripts/legacy-javascript/create-polyfill-size-estimation.js

```diff
@@ -55,7 +55,8 @@ function getPolyfillDependencies() {
   /** @type {Map<string, string[]>} */
   const polyfillDependencies = new Map();
 
-  for (const {name, coreJs3Module} of LegacyJavascript.getPolyfillData()) {
+  for (const {name, modules} of LegacyJavascript.getPolyfillData().filter(d => d.corejs)) {
+    const coreJs3Module = modules[1];
```

Comment:
It isn't super obvious as a consumer of `getPolyfillData` that `modules[0]` is always the coreJs2 module and `modules[1]` is the coreJs3 module.

WDYT about adding a `LegacyJavascript.getCorejsPolyfillData` function that does the `d.corejs` filter and maps to the old item structure?